css/style.css

```diff
@@ -1,19 +1,506 @@
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* Header */
+.header-section {
+  display: flex;
+  flex-flow: row nowrap;
```

Comment:
```suggestion
```
`flex-direction` is `row` by default, and `flex-wrap` is `nowrap` by default, so we don't need this